libraries/auto_generated/data_converter/convertMetro.ts

```diff
@@ -0,0 +1,174 @@
+// To parse this data:
+//
+//   import { Convert, Metro } from "./file";
+//
+//   const metro = Convert.toMetro(json);
+//
+// These functions will throw an error if the JSON doesn't
+// match the expected interface, even if the JSON is valid.
+
+export interface Metro {
+    date:        string;
+    datasets:    Dataset[];
+    labels:      string[];
+    base_period: string;
+}
```

Comment:
Interfaceの定義がキャメルケースに変換されて
```suggestion
export interface Metro {
    date:       string;
    datasets:   Dataset[];
    labels:     string[];
    basePeriod: string;
}
```